apps/aqua/package.json

```diff
@@ -7,8 +7,9 @@
   "scripts": {
     "dev": "tsx watch --clear-screen=false src/index.ts | pino-pretty",
     "bun:dev": "bun run --hot src/index.ts | pino-pretty",
-    "build": "tsup",
-    "start": "node dist/index.js",
+    "deno:dev": "deno run --allow-net --allow-read --allow-env --allow-run --unstable --watch src/index.ts | pino-pretty",
```

Comment:
deno?

Reply:
ahhhhh forgot about that, was testing stuff earlier and forgot to remove that